torch/_refs/__init__.py

```diff
@@ -1654,6 +1655,27 @@ def item(a: TensorLikeType) -> NumberType:
     return number_type(prims.item(a))
 
 
+@register_decomposition(torch.ops.aten.to)
```

Comment:
Let's not do this. If some test that checks that all refs are registered is failing we can add this function to the skip list.